openwisp_monitoring/device/schema.py

```diff
@@ -145,6 +145,74 @@
+                            "mode",
+                            "noise",
+                            "signal",
+                            "ssid",
+                            "tx_power",
+                            "clients",
+                        ],
+                        "properties": {
+                            "channel": {"type": "integer"},
+                            "country": {"type": "string"},
+                            "frequency": {"type": "integer"},
+                            "mode": {"type": "string"},
+                            "noise": {"type": "integer"},
+                            "signal": {"type": "integer"},
+                            "ssid": {"type": "string"},
+                            "tx_power": {"type": "integer"},
+                            "clients": {
+                                "type": "array",
+                                "title": "Wireless Clients",
+                                "additionalItems": False,
+                                "additionalProperties": False,
+                                "required": [
+                                    "aid",
+                                    "assoc",
+                                    "auth",
+                                    "authorized",
+                                    "ht",
+                                    "vht",
+                                    "wds",
+                                    "wmm",
+                                    "wps",
+                                    "mac",
+                                    "mfp",
+                                    "preauth",
+                                    "rrm",
+                                    "signature",
+                                ],
+                                "items": {
+                                    "type": "object",
+                                    "title": "Client",
+                                    "properties": {
+                                        "aid": {"type": "integer"},
+                                        "assoc": {"type": "boolean"},
+                                        "auth": {"type": "boolean"},
+                                        "authorized": {"type": "boolean"},
+                                        "ht": {"type": "boolean"},
+                                        "vht": {"type": "boolean"},
+                                        "wds": {"type": "boolean"},
+                                        "wmm": {"type": "boolean"},
+                                        "wps": {"type": "boolean"},
+                                        "mac": {"type": "string"},
+                                        "mfp": {"type": "boolean"},
+                                        "preauth": {"type": "boolean"},
+                                        "signature": {"type": "string"},
+                                        "rrm": {"type": "array"},
```

Comment:
If you define rrm as an array, you also need to define the type of items.

Please read this guide, it will help you a lot understading jsonschema: https://json-schema.org/learn/getting-started-step-by-step.html

For rrm I would also add maxItems and minItems, both set to 5. Check out https://json-schema.org/understanding-json-schema/reference/array.html for reference